scripts/create-all-tests.js

```diff
@@ -1,12 +1,53 @@
 const path = require('path');
 const fs = require('fs');
-const { createExampleTests } = require('./create-example-tests');
+const {createExampleTests} = require('./create-example-tests');
 
-const scriptDirectory = path.dirname(__filename);
-const rootDirectory = scriptDirectory.split('scripts')[0];
+const args = require('minimist')(process.argv.slice(2), {
+  alias: {
+    h: 'help',
+    d: 'directory',
```

Comment:
Need to update the verbiage where 'directory' actually means 'test plan' from `directory` to `testPlan`